.travis.yml

```diff
@@ -1,28 +1,37 @@
 language: php
 
 php:
-  - 5.6
   - 7.0
```

Comment:
7.0 (and 7.1) are not supported by php anymore

Reply:
yeah, but some people are stuck on outdated infrastructure. as long as its not creating problems, i would prefer to allow older versions.